src/components/popups/MutualsFollowersPopup.vue

```diff
@@ -19,31 +20,18 @@ if (route.name !== `Home`) {
 	}
 }
 
-const authorID = route.name === `Home` ? store.$state.id : route.params.id;
+const authorID = route.name === `Home` ? store.$state.id : (route.params.id as string);
 const profile = computed(() => profilesStore.getProfile(authorID as string));
 // TODO: fetch following from store / backend
-const mutualFollowers = ref<Profile[]>([
-	{
-		id: `oiahefoiheoafheaf`,
-		name: `Tom Brady`,
-		email: `tb12@gmail.com`,
-		bio: `6-time super bowl champion`,
-		location: `Tampa Bay`,
-		avatar: ``,
-		socials: [],
-		website: `tb12.com`,
-	},
-	{
-		id: `fziohogheabfhoeaof`,
-		name: `Tom Not Brady`,
-		email: `tb12@gmail.com`,
-		bio: `6-time super bowl champion`,
-		location: `Tampa Bay`,
-		avatar: ``,
-		socials: [],
-		website: `tb12.com`,
-	},
-]);
+const mutuals = ref<Set<string> | undefined>();
+
+onBeforeMount(async () => {
+	const myC = await connections.fetchConnections(store.id);
+	const theirC = await connections.fetchConnections(authorID);
+	if (myC && theirC) {
```

Comment:
We can change this as well similar to the way we're using it in line 39 of `MutualFollowersWidget`